etc/compose/cdh5.15-hive/docker-compose.yml

```diff
@@ -2,5 +2,5 @@ version: '2.0'
 services:
```

Comment:
I would not rename, but copy. So we could have two versions for a while.

Reply:
I would rename, not copy. I don't see a need for 2 versions (also, cdh5.13:6 won't disappear, we can use that for as long as we want)